src/lesson6_task1/Transport.java

```diff
@@ -0,0 +1,46 @@
+package lesson6_task1;
+import java.nio.charset.StandardCharsets;
+import java.util.Locale;
+
+import java.util.Scanner;
+
+public abstract class Transport {
+    int powerHours;
+    float powerKW;
+    int maxSpeed;
+    float weight;
+    String brand;
+    String technicalChar = "";
+    static Scanner scanner = new Scanner(System.in, StandardCharsets.UTF_8);
+
+    //конструктор
+    public Transport(int powerHours, int maxSpeed, String brand, float weight) {
+        this.powerHours = powerHours;
+        this.powerKW = powerKW;
+        this.maxSpeed = maxSpeed;
+        this.weight = weight;
+        this.brand = brand;
+        this.technicalChar = technicalChar;
```

Comment:
powerKW и technicalChar не передаются в конструктор. Что ты тогда присввамваешь этим значениям на объекте?
```suggestion
    public Transport(int powerHours, int maxSpeed, String brand, float weight) {
        this.powerKW = powerKW;
        this.maxSpeed = maxSpeed;
        this.weight = weight;
        this.brand = brand;
```

>###  _В условии для Transport только 4 обязательных поля, а не 6!_ <br><p>&#160;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;↓<p>


- Мощность(в лошадиных силах)
- Максимальная скорость(км/ч
- Масса (кг)
- Марка( например: Audi, BMW , Boing, Airbus, Scania , МАЗ и т.д. )

Reply:
Исправила, оставила только 4 поля как в задании.